app/controllers/users.js

```diff
@@ -0,0 +1,20 @@
+const services = require('../services/user');
+const log = require('../logger');
+
+exports.addUser = (req, res, next) =>
+  services
+    .createUser(req.body)
+    .then(result => {
+      log.info(`User created first name: ${result.firstName}, last name: ${result.lastName}`);
+      return result;
+    })
+    .then(result =>
+      res
+        .json(result)
```

Comment:
warning: you are returning the user created in a POST request. Return only a 201 status.
201 status often indicates that something was created.